packages/api/src/sequelize/migrations/2024-04-18_00_update-facility-organization.ts

```diff
@@ -0,0 +1,98 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+import { FacilityType } from "../../models/medical/facility";
+import { OrganizationType } from "../../models/medical/organization";
+
+const facilityTableName = "facility";
+const organizationTableName = "organization";
+
+// Use 'Promise.all' when changes are independent of each other
+// Docs: https://sequelize.org/api/v6/class/src/dialects/abstract/query-interface.js~queryinterface
+export const up: Migration = async ({ context: queryInterface }) => {
+  return queryInterface.sequelize.transaction(async transaction => {
+    await queryInterface.addColumn(
+      facilityTableName,
+      "oid",
+      { type: DataTypes.STRING },
+      { transaction }
+    );
```

Comment:
We should prob add a `unique` constraint to this column to avoid collisions, eh?